components/molecules/FavoriteRepoCard/favorite-repo-card.tsx

```diff
@@ -0,0 +1,42 @@
+import Avatar from "components/atoms/Avatar/avatar";
+import { truncateString } from "lib/utils/truncate-string";
+import { StaticImageData } from "next/image";
+import Link from "next/link";
+import React from "react";
+
+export interface FavoriteRepoCardProps {
+  avatarURL?: string | StaticImageData;
+  name?: string;
+  owner?: string;
+  topic?: string;
+  userPage?: string;
```

Comment:
Small change here, technically only the `userPage` can be optional since all the other props are required data in `DbRepo` global entity - it doesn't make sense to make the component more loose unless it is designed to be able to render other entities than repositories; this should also make the conditionals below easier and disable default values at the same time 👍

Reply:
👍